components/ModPagination.vue

```diff
@@ -0,0 +1,119 @@
+<template>
```

Comment:
I just realized this is the old file. Let's delete this one and we're good to go!